dhis-2/dhis-services/dhis-service-dxf2/src/main/java/org/hisp/dhis/dxf2/events/trackedentity/store/mapper/EventRowCallbackHandler.java

```diff
@@ -67,8 +67,8 @@ private Event getEvent( ResultSet rs )
         event.setEvent( rs.getString( getColumnName( COLUMNS.UID ) ) );
         event.setId( rs.getLong( getColumnName( COLUMNS.ID ) ) );
         event.setTrackedEntityInstance( rs.getString( getColumnName( COLUMNS.TEI_UID ) ) );
-        final String followup = rs.getString( getColumnName( COLUMNS.ENROLLMENT_FOLLOWUP ) );
```

Comment:
This returns "t" and Boolean.parseBoolean( "t" ) is false. Changed it to use resultSet.wasNull() to preserve null